samza-core/src/main/java/org/apache/samza/config/ClusterManagerConfig.java

```diff
@@ -280,4 +289,8 @@ public boolean getJmxEnabledOnJobCoordinator() {
       return true;
     }
   }
+
+  public int getCoordinatorUrlPort() {
```

Comment:
Can we document that this is for experimental purpose and evolving, therefore, backward compatibility is not guaranteed and maybe removed any time.

Reply:
Updated the javadoc to reflect this. Thanks for the suggestion.